runtime/src/accounts_db.rs

```diff
@@ -6990,6 +7003,62 @@ impl AccountsDb {
                 })
                 .sum();
 
+            // subtract data.len() from accounts_data_len for all old accounts that are in the index twice
+            if pass == 0 {
```

Comment:
For the "remove duplicates" block, I first put the `if pass == 0 {}` around it.

I also applied the same tweaks of "compute sums locally without atomics, then reduce at the end with Atomics"